src/gui/relationships.ts

```diff
@@ -133,13 +132,13 @@ export class GuiRelationships extends GuiSubscreen {
 			const allow = e.memberNumber === Player.MemberNumber ? this.relationshipsData.access_modify_self : this.relationshipsData.access_modify_others;
 
 			// Relationships list
-			MainCanvas.strokeRect(130, Y, 700, 64);
+			DrawEmptyRect(130, Y, 700, 64, "white");
```

Comment:
The default stroke color is `Black` and width should be 2 here.
```suggestion
			DrawEmptyRect(130, Y, 700, 64, "Black", 2);
```